dev/ts/generate

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -e
+
+(
+  cd dev/ts/protoc
+  docker build -t xmtp/protoc .
```

Comment:
We don't use docker for the Go side, does this make it easier?

Reply:
I _could_ use it for both. The go side runs the generator in a pre-recommit hook, so I'm trying to keep that flow as lean as possible. Given how rarely this changes, was also thinking of breaking this out into a tiny repo of its own so that we don't have to build the image on its own.